compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -368,6 +368,10 @@ public void visit(BLangAnnotation annotationNode) {
         }
     }
 
+    private boolean isNullOrEmpty(String s) {
```

Comment:
Do we need to have a separate method for this simple check? 
Also, both times we are calling the negation of the method `!isNullOrEmpty()` would it make sense if we change it to `containsValue()` or something similar if we are having a separate method?

Reply:
I personally find it easier to glans over it when these conditions have names.

We can simply swap the if block and else block to get rid of `!` but it may hinder the thinking flow of the reader, I guess it's a personal preference thing.